### PR TITLE
Added boxplot

### PR DESCRIPTION
**Pull request by rrrohit1**
- Removed old movie genres
- Removed bins (not required)
- Removed null from ratings

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -475,4 +467,4 @@ def update_output(cat, rate, year):
 
 
 if __name__ == '__main__':
-    app.run_server(debug = True)
\ No newline at end of file
+    app.run_server(debug = True)
```
Comment (thayeylolu) [inline src/app.py:470]: I would suggest we remove the debug option

**Review by thayeylolu — APPROVED** [review]
I would suggest we remove the debug option.

File: src/app.py (hunk 1 above)
Comment (rrrohit1) [reply src/app.py:470]: Let us remove it in the final merge to main branch.

Comment (rrrohit1) [reply src/app.py:470]: Can you merge this PR? @thayeylolu

**Pull request merged**